Dockerfile

```diff
@@ -4,7 +4,9 @@ RUN apt update && apt upgrade -y
 
 ARG PORT=5000
 ARG APP_ENV=production
+ARG RACK_ENV=production
```

Comment:
I see the usecase for the `RACK_ENV` as NewRelic seems to expect `RACK_ENV` or `RAILS_ENV` - however, is it not possible to either:
1. Define only one `ARG` as `ENV=production` and reuse both places?
2. Instead of using `APP_ENV`, replace all uses with `RACK_ENV`?

I see a case where one might build and override one of the two variables with another environment but not the other causing weird discrepancies by mistake?
https://docs.docker.com/reference/dockerfile/#arg

Reply:
We could add a build arg that sets both of the variables, sure. The problem is that Sinatra seems to expect APP_ENV and NewRelic RACK_ENV  - and this is not something we can change on our side, so both seemingly need to be set. I am not 100% sure about this.
Personally, I don't mind having both as args since they just default to production, and any production-related artifact is generated by the pipeline anyway, so not something done by hand.
But either way is fine

Reply:
But running tests and hosting the app locally (i.e. in development mode) would require us to set both values to the same?

Reply:
Discussed in person that Sinatra expects environment variable which means we will update multiple places nonetheless